charts/argo-cd/Chart.yaml

```diff
@@ -28,3 +28,5 @@ annotations:
   artifacthub.io/changes: |
     - kind: added
       description: Added priorityClassName for the redis secret init job
```

Comment:
Can you remove the lines from the previous change? (Only the current change [yours] needs to exists inside this structure)

Reply:
Yes, done!